File: Answer/3718.缺失的最小倍数.py
#
# @lc app=leetcode.cn id=3718 lang=python3
#
# [3718] 缺失的最小倍数
#

# @lc code=start
from typing import List

class Solution:
    def missingMultiple(self, nums: List[int], k: int) -> int:
        s = set(nums)
        multiple = k
        while multiple in s:
            multiple += k
        return multiple
# @lc code=end

File: Answer/3718.缺失的最小倍数.js
/*
 * @lc app=leetcode.cn id=3718 lang=javascript
 *
 * [3718] 缺失的最小倍数
 */

// @lc code=start
/**
 * @param {number[]} nums
 * @param {number} k
 * @return {number}
 */
var missingMultiple = function (nums, k) {
  const set = new Set(nums);
  let multiple = k;
  while (set.has(multiple)) multiple += k;
  return multiple;
};
// @lc code=end
